packages/fuselage-ui-kit/src/blocks/VideoConferenceBlock/VideoConferenceBlock.tsx

```diff
@@ -97,6 +101,23 @@ const VideoConferenceBlock = ({
     }
   };
 
+  const getMessageFooterText = useCallback(
```

Comment:
I don't think this has to be a useCallback, since it's just called below, it won't trigger rendering.

Maybe instead we could have a useMemo, or define this function outside the component?